test_Areg_Avetisyan.py

```diff
@@ -0,0 +1,2 @@
+def test_print_name(name="Areg"):

```

Comment:
Change your function.
It should take an input from user 3 parameters: name, surname, age and print inputs.